src/med_bench/utils/classifiers.py

```diff
@@ -0,0 +1,508 @@
+from itertools import combinations
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+from numpy.random import default_rng
+from scipy import stats
+from scipy.special import expit
+from scipy.stats import bernoulli
+from sklearn.base import clone
+from sklearn.calibration import CalibratedClassifierCV
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.linear_model import LassoCV, LogisticRegressionCV, RidgeCV
+from sklearn.model_selection import KFold
+from sklearn.preprocessing import PolynomialFeatures
+
+from .utils import _convert_array_to_R, _get_interactions
+
+ALPHAS = np.logspace(-5, 5, 8)
+CV_FOLDS = 5
+TINY = 1.e-12
+
+def _get_x_classifiers(regularization, forest, calibration, calib_method):
```

Comment:
naming is bad, if we predict t, should be t_classifier? or t_x_classifier ?

Reply:
This has been addressed in 3f1b34577643b37b886202ab5873e7b5a9054a40